src/riak_core_vnode_worker.erl

```diff
@@ -38,11 +38,12 @@
         modstate :: any()
```

Comment:
If there's anything else, let me know. Otherwise I'll rebase all these fixup commits back into the original checkin.